x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/esql/discover_timeline_state_integration.cy.ts

```diff
@@ -212,6 +203,7 @@ describe.skip(
         cy.wait(`@${TIMELINE_REQ_WITH_SAVED_SEARCH}`);
         openKibanaNavigation();
         navigateFromKibanaCollapsibleTo(STACK_MANAGEMENT_PAGE);
+        confirmLeaveUnsavedTimeline();
         cy.get(LOADING_INDICATOR).should('not.exist');
         goToSavedObjectSettings();
         cy.get(LOADING_INDICATOR).should('not.exist');
```

Comment:
should we extract the line below into a task function? (`cy.get(SAVED_OBJECTS_TAGS_FILTER).trigger('click');`)